### DISCUSSION TITLE
Como usar o GitHub para organizar seu dia a dia

### DISCUSSION THREAD

**Discussion by mlarasusan** [gittogethers/octogatosconf22#103] [Talks in Portuguese]
**Palestrante:** Roberson Miguel dos Santos

**Assunto**
Muitas vezes nos percebemos com um acúmulo de tarefas a serem realizadas, com nossas mentes lotadas e anotações espalhadas.

Ajudando menores carentes a iniciarem em tecnologia, optamos por usar o Github para dar uma ideia de organização e prioridade possível e isso deu muito certo.

**Comment by morgannadev** [comment]
Amei demais o conteúdo, foi sensacional. Deixo uma sugestão pra galera que estiver montando o quadro com as orientações do Roberson de ter uma coluna de impedimentos.  Às vezes, algumas tarefas tem algumas travas que ficam difíceis de visualizar. Ao menos para mim, ajuda bastante ter uma coluna separada só para isso. Assim posso reservar um momento do meu dia para verificar o que é possível resolver ou o que ainda continua travado.

Parabéns demais, Roberson, adorei a palestra!

**Reply by roberson-miguel** [reply]
Perfeito Morgana. Bem interessante a solução da coluna impedimentos. Eu usava os comentários mesmos.

**Comment by renata-amorim** [comment]
Olá Roberson, muito obrigada pela palestra, eu amei, estava precisando.
Preciso organizar meus estudos, estou fazendo migrando de carreira de Nutrição para a área de tecnologia, mais precisamente front-end e tenho muita coisa para estudar, tem dias que fico muito perdida sem saber por onde começar e acabo adiando muito o que preciso fazer, muito obrigada, seus slides estavam lindo também. 

Parabéns!

**Reply by roberson-miguel** [reply]
Obrigado.

**Comment by bpires** [comment]
Pensando nos conceitos/recursos do GitHub apresentados pelo Roberson, deixo aqui a forma como eu gosto de sistematizá-los, se ajudar alguém:

```
.
├──	Projects = conjuntos de milestones 
│  		└── Milestones = conjuntos de issues
│	   			└── Issues = conjuntos de tasks
│						└── Tasks = o menor divisão possível de uma atividade em passos
│
└── Labels (extra): tags para reorganizar suas issues não hierarquizadamente.
```

Foi assim que entendi, mas se alguém tiver alguma correção/acréscimo, agradeço ✨

**Reply by roberson-miguel** [reply]
Ótima forma de organizar os pensamentos. Obrigado por compartilhar

**Comment by roberson-miguel** [comment]
Deixar aqui o link do repositório com o modelo de organização proposto e nele tem o pdf da palestra.

https://github.com/roberson-miguel/organizar-dia-a-dia

**Comment by roberson-miguel** [comment]
> **Palestrante:** Roberson Miguel dos Santos
> 
> **Assunto**
> Muitas vezes nos percebemos com um acúmulo de tarefas a serem realizadas, com nossas mentes lotadas e anotações espalhadas.
> 
> Ajudando menores carentes a iniciarem em tecnologia, optamos por usar o Github para dar uma ideia de organização e prioridade possível e isso deu muito certo.

Link 

https://github.com/roberson-miguel/organizar-dia-a-dia
